tests/resources/helper_functions.py

```diff
@@ -317,7 +308,7 @@ def convert_bt_amt_to_dt(
 
     Datatokens always have 18 decimals, even when the base tokens don't.
     """
-    unit_value = Decimal(10) ** 18  ## FIXME: SHOULDN'T `18` BE `bt_decimals` ??
+    unit_value = Decimal(10) ** 18  # FIXME: SHOULDN'T `18` BE `bt_decimals` ??
```

Comment:
@trentmc there's a comment right above, we are using the datatoken decimals, not bt_decimals. Also, it's a test helper, so we can allow more presumption

Reply:
Something smells in that code. And I had other issues in test_exchange_fees.py. To make sure we get to the heart of the matter, I just created a new github issue for it.
https://github.com/oceanprotocol/ocean.py/issues/1190